src/clients/save_network.rb

```diff
@@ -247,15 +247,27 @@ def copy_from_instsys
       nil
     end
 
+    # Creates DNS configuration on target
+    def configure_dns
+      if Mode.autoinst
+        NetworkAutoYast.instance.configure_dns
```

Comment:
Regarding the several sets of configuration data that YaST deals with, this method assigns the "AY profile" to the "YaST working set", but does not write it to the "persistent system config" or "current system config".
OTOH NetworkAutoconfiguration::configure_dns does write to the system.
It is inconsistent, and seems to contradict the comment above, "Creates DNS configuration on target".